client/src/components/ElementEditor/Element.js

```diff
@@ -1,33 +1,52 @@
 import React, { PureComponent } from 'react';
 import { elementType } from 'types/elementType';
+import { inject } from 'lib/Injector';
 
 /**
  * The Element component used in the context of an ElementEditor shows the summary
  * of an element's details when used in the CMS, including ID, Title and Summary.
  */
 class Element extends PureComponent {
   render() {
-    const { element: { ID, Title, Summary } } = this.props;
+    const { element: { ID, Title, Summary, Type, IconClass },
+      HeaderComponent,
+      ContentComponent } = this.props;
```

Comment:
I'm not a huge fan of this formatting, how would this be instead?

```js
const {
  element: { ID, Title, Summary, Type, IconClass },
  HeaderComponent,
  ContentComponent,
} = this.props;
```

Reply:
sure, will change.